Tasks/Common/packaging-common/nuget/NuGetToolGetter.ts

```diff
@@ -110,13 +110,20 @@ function pathExistsAsFile(path: string) {
 // Based on code in Tasks\Common\MSBuildHelpers\msbuildhelpers.ts
 export async function getMSBuildVersionString(): Promise<string> {
     const msbuild2019Path = 'C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/MSBuild/Current/Bin/msbuild.exe';
+    const msbuild2022Path = 'C:/Program Files/Microsoft Visual Studio/2022/Enterprise/MSBuild/Current/Bin/msbuild.exe';
```

Comment:
Is there a guarantee that msbuild will always be in that path?

Reply:
That is the location for the VS 2022 Enterprise that is used in the Windows 2022 hosted agent. I also have the enterprise installed in that location but I suppose if you have the the VS 2022 Professional, the Enterprise will be replaced with Professional in the path. But we don't support that scenario for msbuild2019 so I wonder if it's necessary for 2022